ctrlx-config.js

```diff
@@ -70,7 +70,10 @@ module.exports = function(RED) {
             return res.end(err.message);
           }
         })
-        .finally(() => ctrlx.logOut());
+        .finally(() => {
+          // We have to catch, because this fails for bad credentials
+          ctrlx.logOut().catch((err) => console.log('Failed to log out with error ' + err.message));
```

Comment:
Can you please check if RED.log.warn() would work here? I think, this would be the better option than logging directly to console. But I'm not sure if it works in this context. Also Message "Failed to log out when trying to browse with error" gives more information about what happened.